docker/docker-compose.yml

```diff
@@ -17,7 +20,8 @@ services:
       - VMPOOLER_DEBUG=true # for use of dummy auth
       - VMPOOLER_CONFIG_FILE=/etc/vmpooler/vmpooler.yaml
       - REDIS_SERVER=redislocal
-      - LOGFILE=/dev/stdout
+      - LOGFILE=/dev/null
```

Comment:
No logs?

Reply:
For some reason the logs get duplicated in my docker compose environment if I set this logfile to be stdout. I found that when I set it to be /dev/null that I got the log messages, but they were no longer duplicated. Wonky stuff. I can remove this if needed.

Reply:
That's pretty strange. I guess we don't really need logs in the docker compose environment beyond what goes to stdout already so it's probably fine.